jgrapht-core/src/main/java/org/jgrapht/generate/GnmRandomBipartiteGraphGenerator.java

```diff
@@ -194,6 +197,22 @@ public void generateGraph(
 
     }
 
+    /**
+     * Returns the first partition of vertices in the bipartite graph.
+     * @return one partition of the bipartite graph
+     */
+    public Collection<V> getFirstPartition(){
```

Comment:
This should return `Set<V>`. Use something like new `HashSet<>(partitionA.values())`.